meta.yaml

```diff
@@ -1,26 +1,25 @@
-{% set version = "2020.7.30.1" %}
+{% set version = "2021.07.0" %}
 
 package:
   name: geocat-viz
   version: {{ version }}
 
 source:
-  url: https://github.com/NCAR/geocat-viz/archive/v{{ version }}.tar.gz
+  git_rev: main # https://github.com/NCAR/geocat-viz/archive/v{{ version }}.tar.gz
+  git_url: https://github.com/ncar/geocat-viz.git
 
 build:
-  noarch: python
```

Comment:
Is there a specific reason for removing `noarch`?

Reply:
Not that I'm aware of. I believe this was a change made by Alea. 

@pilotchute, what does this do?